Crashlytics/Crashlytics/Components/FIRCLSApplication.m

```diff
@@ -47,9 +49,9 @@
 #elif TARGET_OS_TV
   return @"tvos";
 #elif TARGET_OS_WATCH
-  return @"ios";  // TODO: temporarily use iOS until Firebase can add watchos to the backend
+  return @"ios";
 #elif defined(TARGET_OS_VISION) && TARGET_OS_VISION
-  return @"ios";  // TODO: temporarily use iOS until Firebase can add visionos to the backend
```

Comment:
Were these TODOs intentionally removed?

Reply:
Yes, Follow back on https://github.com/firebase/firebase-ios-sdk/pull/11938#discussion_r1358798948, indeed it is a legacy function, Crashlytics should have proper support for watchOS, but we need to followup on backend change for VisionOS with team later.

Reply:
The TODOs are not valid any more.